packages/ruby-babeltrace2/package.py

```diff
@@ -10,6 +10,8 @@ class RubyBabeltrace2(RubyPackage):
     homepage = "https://github.com/argonne-lcf/babeltrace2-ruby"
     url      = "https://rubygems.org/downloads/babeltrace2-0.1.1.gem"
 
+    version('0.1.3', sha256='549cbd03c6e4987cb935ee20ca69887f8fd6bd5d2e6d9fca0706b63d966fca5c', expand=False)
+    version('0.1.2', sha256='a09696933a5d36b2833bd6e3bda5b3981d9719ee77c71a5542ccc127e02b27c3', expand=False)
     version('0.1.1', sha256='2a5c35a72ded62240230dd5b31eb7f2e6cdeabc5b29685a0b636c73c17a6c30c', expand=False)
```

Comment:
Why the sha256 here doesn't match the one in https://rubygems.org/gems/babeltrace/versions/0.1.3? 
` bce6133637c18d503efb6b88ebffcbbdef8561310366b5c5d32751f6ae4b055c `

Reply:
Here is what I did (and redid):
```sh
wget https://rubygems.org/downloads/babeltrace2-0.1.3.gem
sha256sum babeltrace2-0.1.3.gem
549cbd03c6e4987cb935ee20ca69887f8fd6bd5d2e6d9fca0706b63d966fca5c  babeltrace2-0.1.3.gem
```
Could you  try and tell me what you get?

Reply:
> Why the sha256 here doesn't match the one in https://rubygems.org/gems/babeltrace/versions/0.1.3? `bce6133637c18d503efb6b88ebffcbbdef8561310366b5c5d32751f6ae4b055c`

Because this is babeltrace you're looking at, not babeltrace2